### ISSUE TITLE
Symfony new reports wrong PHP version installed

### ISSUE THREAD

**Issue by HigorOliveira056** [symfony-cli/symfony-cli#326]
I tried to create a new project with the symfony CLI but it says that it is not possible to create it, because my PHP version is not 8.1, but this is the version I have installed

![image](https://github.com/symfony-cli/symfony-cli/assets/59327952/4647dfab-aa13-49e6-b104-c219edf23abe)

**Comment by HigorOliveira056** [comment]
apparently it is getting the PHP version located in my installed XAMPP instead of the one configured in the environment variables
![image](https://github.com/symfony-cli/symfony-cli/assets/59327952/82192612-9841-4737-9817-6c98a15e5529)

the interesting thing is that PHP 8.1 is configured in environment variables instead of 7

**Comment by tucksaun** [comment]
Is your PHP 8 install detected by Symfony CLI (what is the output of `symfony local:php:list`) ?

**Comment by HigorOliveira056** [comment]
yes, I solved it using .phpversion in the project because symfony was tracking both versions the interesting fact is that I don't know why it lists php 7
